src/core/ui/list/ItemView.tsx

```diff
@@ -8,6 +8,21 @@ interface ViewProps {
   visualRight?: ReactElement<{ sx: { flexShrink: number } }>;
 }
 
+const cloneVisual = <Sx extends { flexShrink: number }>(element: ReactElement<{ sx: Partial<Sx> }> | undefined) => {
+  if (!element) {
+    return element;
+  }
```

Comment:
I'm not sure but this looks counterintuitive. I liked it better with  visual && cloneElement...

Reply:
Can we put return undefined?

Reply:
Yes, that might be the best option in my opinion.